### DISCUSSION TITLE
Game/File Sharing

### DISCUSSION THREAD

**Discussion by AJFord21** [OnionUI/Onion#1574] [Ideas]
Hi Team! 

Really REALLY love your work-- exceptionally well built OS.

Thought it'd be cool to be able to set up a local network and share ROMs across machines. I'd love to be able to quickly ship a file to my kids or friends without having to money with the SD card.

Might be foolish, technically too challenging or whatever but thought I'd throw it out there.

Please keep up the wonderful work! We appreciate you! 

-AJ

**Comment by thomasklemenc** [comment]
Hi AJ,

providing functionality that could facilitate sharing roms in any way is absolutely no-go and probably would get us in trouble, sorry.

Thanks for the praise ❤️
